### ISSUE TITLE
[BUG] State.locs_to_include out of bounds

### ISSUE THREAD

**Issue by lambdasapple** [farkon00/cont#41]
Compiling the following code
```
const SYSEXIT 60 end

proc exit int: SYSEXIT syscall1 drop end 

1 exit
```
will lend in the following error message

```
Traceback (most recent call last):
  File "/home/datawater/Desktop/Programming/third-party/cont/playground/../cont", line 99, in <module>
    main()
  File "/home/datawater/Desktop/Programming/third-party/cont/playground/../cont", line 94, in main
    compile_ops(ops)
  File "/home/datawater/Desktop/Programming/third-party/cont/generating/generating.py", line 16, in compile_ops
    return TARGETS[State.config.target](ops)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/datawater/Desktop/Programming/third-party/cont/generating/fasm_x86_64_linux.py", line 36, in compile_ops_fasm_x86_64_linux
    f.write(generate_fasm_x86_64_linux(ops))
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/datawater/Desktop/Programming/third-party/cont/generating/fasm_x86_64_linux.py", line 130, in generate_fasm_x86_64_linux
    buf += generate_op_fasm_x86_64_linux(op)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/datawater/Desktop/Programming/third-party/cont/generating/fasm_x86_64_linux.py", line 328, in generate_op_fasm_x86_64_linux
    return comment + generate_operator_fasm_x86_64_linux(op)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/datawater/Desktop/Programming/third-party/cont/generating/fasm_x86_64_linux.py", line 636, in generate_operator_fasm_x86_64_linux
    f"mov r12, {len(State.locs_to_include[op.loc_id]) + 1}\n"
                    ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
IndexError: list index out of range
```

Command used to compile `python3 cont minimal.cn`

Uname: `Linux farkonisaboykisser 6.6.6-200.fc39.x86_64 #1 SMP PREEMPT_DYNAMIC Mon Dec 11 17:29:08 UTC 2023 x86_64 GNU/Linux`

**Comment by lambdasapple** [comment]
Fixed by 6d89de1

**Comment by farkon00** [comment]
Fixed in commit [6d89de1](https://github.com/farkon00/cont/commit/6d89de144c53673d60efa235cf7bd59169326f70) 3 minutes before issue submission via mind-reading abilities of the maintainers.
